api/src/main/proto/stellarstation/api/v1/radio/radio.proto

```diff
@@ -43,6 +43,9 @@ message RadioDeviceConfiguration {
   // The protocol used by this device when doing RF communication. If unset, the device is only
   // demodulating / modulating without applying any higher-level communication protocol.
   CommunicationProtocol protocol = 4;
+
+  // The polarization used by this radio device.
+  Polarization polarization = 5;
```

Comment:
@rtubio 
- Can vertical and horizontal polarization be used together?
- What does "simultaneous LHCP/RHCP" actually mean? 
Is it "LHCP and RHCP" or  "LHCP or RHCP", or a special polarization that combine LHCP and RHCP?

I am asking the question because I would like to know if we should define polarization  as an array, or single value.